### ISSUE TITLE
Ordering of commands

### ISSUE THREAD

**Issue by Beanow** [benmatselby/sublime-phpcs#126]
The ordering seems to cause some issue.
For example:
```json
{
    "phpcs_sniffer_run": true,
    "phpcs_command_on_save": false,
    "phpcs_show_errors_on_save": false,
    "php_cs_fixer_on_save":true,
    "phpmd_run": true,
    "phpmd_command_on_save": true
}
```

This will add the mess detector markers, but remove them shortly after if the cs fixer makes any changes to the file. Ideally some form of ordering is available to be able to customize this behaviour.

**Comment by benmatselby** [comment]
@Beanow Hi there

I've just been looking into this and it's an interesting one. Technically the php-cs-fixer is running first, from what I can see, but the sniffers run after but don't see the fixer change.. I see the opposite of you. So I actually see the markers in the gutter, but they are wrong, because the fixer has fixed the issues.

So are you wanting to run the fixer, and then see what sniffs are found afterwards?

**Comment by Beanow** [comment]
Yes, I think the fixer settings are easy to define and predictable. I would expect it to 'silently' fix them and sniff afterwards.

Ben Selby <notifications@github.com> schreef op 24 augustus 2015 22:05:27 CEST:
